### ISSUE TITLE
Azure Machine Learning - Remote extension does not work on WSL

### ISSUE THREAD

**Issue by gro1m** [microsoft/vscode-tools-for-ai#1982]
Does this occur consistently? 
Repro steps:

1.
2.

Action: azureml.viewContext.computeInstance.connect
Error type: 412
Error Message: Unable to install Azure Machine Learning - Remote extension, please retry installing from VS Code Extensions view. Operation timed out.

Version: 0.28.0
OS: linux
OS Release: 5.10.102.1-microsoft-standard-WSL2
Product: Visual Studio Code
Product Version: 1.76.2
Language: en

<details>
<summary>Call Stack</summary>

```
extension.js:2:2098745
```

</details>

Further information:
```bash
~/home$ code --install-extension ms-toolsai.vscode-ai-remote
Installing extensions on WSL: Ubuntu-20.04...
Cannot install the 'ms-toolsai.vscode-ai-remote' extension because it is declared to not run in this setup.
```

So not sure this is supported, but we need it.

**Comment by gro1m** [comment]
Related but not same environment: https://github.com/microsoft/vscode-tools-for-ai/issues/1981

**Comment by tbombach** [comment]
Hi @gro1m. We haven't tested using the remote extension in WSL as a scenario that we explicitly supported. I will look into it more to see if we can get this scenario working for you.

**Comment by shsuman** [comment]
After doing some preliminary testing, it seems to work by simply tweaking the `extensionKind` to `["workspace", "ui"]` in `package.json`. If there are no side effects, we can enable it. I will create an issue to Enable Azure ML - Remote extension in workspace to fully test for any potential side effects before enabling it.

**Comment by shsuman** [comment]
Closed in favor of #2056
